.github/workflows/tests-integration.yml

```diff
@@ -18,7 +18,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.6, 3.7, 3.8, 3.9]
+        python-version: ["3.6", "3.7", "3.8", "3.9", "3.10"]
```

Comment:
Better to use single quotation to align other lines

Reply:
Thank you for your suggestion. I've updated all `python-version` in this PR.